src/debug_menu.cpp

```diff
@@ -151,7 +158,9 @@ static int info_uilist()
         uilist_entry( DEBUG_SHOW_MSG, true, 'd', _( "Show debug message" ) ),
         uilist_entry( DEBUG_CRASH_GAME, true, 'C', _( "Crash game (test crash handling)" ) ),
         uilist_entry( DEBUG_DISPLAY_NPC_PATH, true, 'n', _( "Toggle NPC pathfinding on map" ) ),
-        uilist_entry( DEBUG_TEST_WEATHER, true, 'W', _( "Test weather" ) )
+        uilist_entry( DEBUG_TEST_WEATHER, true, 'W', _( "Test weather" ) ),
+        uilist_entry( DEBUG_SAVE_SCREENSHOT, true, 'H', _( "Take Screenshot" ) ),
+        uilist_entry( DEBUG_GAME_REPORT, true, 'r', _( "Generate Game Report" ) ),
```

Comment:
Please decapitalize this and screenshot above.

Reply:
Will do! Thank you for your feedback :+1: